corehq/ex-submodules/auditcare/tests/test_gdpr_scrub_user_auditcare.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+from django.test import TestCase
+from django.core import management
+from auditcare.utils.export import get_auditcare_docs_by_username
+from auditcare.models import NavigationEventAudit
+from dimagi.utils.couch.database import iter_bulk_delete
+
+
+class TestGDPRScrubUserAuditcare(TestCase):
+
+    def setUp(self):
+        NavigationEventAudit(user="test_user1", request_path="/fake/path/0").save()
+        NavigationEventAudit(user="test_user1", request_path="/fake/path/1").save()
+        NavigationEventAudit(user="test_user1", request_path="/fake/path/2").save()
+        self.returned_docs = get_auditcare_docs_by_username("test_user1")
```

Comment:
I don't think you use this anywhere

Reply:
Removed here:
https://github.com/dimagi/commcare-hq/pull/19972/commits/c1295fe519e00d372639d089978884bf7587ad3f